scripts/tmux-ssh-split.sh

```diff
@@ -406,18 +406,19 @@ extract_path_from_ps1() {
       return 0
     fi
 
+    # Replace ~ with $HOME to avoid no such file or directory error. See issue #16
+    match="${match/\~/\$HOME}"
```

Comment:
This expands on the host executing it... It would only produce effective results if you happen to run this on a host where HOME is the same as on the remote.

Reply:
It is escaped to make sure it does not expand on the host. The ssh command was already being sent in `'` so the expansion happens on the remote. You can test it by `ssh some-other-user@remote -t 'echo "$USER"` and it will print "some-other-user" and not the local user. You are alrady using it in the ssh command - see `$SHELL` in `ssh ... ; exec ${SHELL:-/usr/bin/env sh} -l'` The variable `$SHELL` is not expanding on the host because of single quotes.

Reply:
oh I completely misread that and overlooked the `\`. It was a very late review :)

Reply:
I included this change in 168917eb5e0d2c7eae73207162761ffb3906bf93
Thanks!